content/applications/general/email_communication/email_domain.rst

```diff
@@ -1,163 +1,178 @@
    `include:_spf.odoo.com`: `v=spf1 include:_spf.odoo.com include:_spf.google.com ~all`
 
-You can check if your SPF record is valid with a free tool like 
-`MXToolbox SPF <https://mxtoolbox.com/spf.aspx>`_.
+Check if the SPF record is valid with a free tool like `MXToolbox SPF <https://mxtoolbox.com/spf.
+aspx>`_.
 
 .. _email_communication/DKIM_compliant:
 
 Enable DKIM
 ===========
 
-The DomainKeys Identified Mail (DKIM) allows you to authenticate your emails with a digital signature.
+The Domain Keys Identified Mail (DKIM) allows a user to authenticate emails with a digital
+signature.
 
 When sending an email, the Odoo server includes a unique DKIM signature in the headers. The
-recipient's server decrypts this signature using the DKIM record in your domain name. If the
-signature and the key contained in the record match, this guarantees that your message is authentic
-and has not been altered during transport.
+recipient's server decrypts this signature using the DKIM record in the database's domain name. If
+the signature and the key contained in the record match, this guarantees that the message is
+authentic and has not been altered during transport.
 
-To enable DKIM, you must add a CNAME record to the DNS zone of your domain name:
+To enable DKIM, add a CNAME record to the DNS zone of the domain name:
 
-``odoo._domainkey IN CNAME odoo._domainkey.odoo.com.``
+`odoo._domainkey IN CNAME odoo._domainkey.odoo.com.`
 
 .. tip::
-   If your domain name is `mycompany.com`, you need to create a subdomain 
+   If the domain name is `mycompany.com`, make sure to create a subdomain
    `odoo._domainkey.mycompany.com` whose canonical name is `odoo._domainkey.odoo.com.`.
 
-How to create or modify a CNAME record depends on the provider hosting the DNS zone of your domain
-name. The most common providers are list below.
+The way to create or modify a CNAME record depends on the provider hosting the DNS zone of the
+domain name. The most common providers are
+:ref:`listed below <email_communication/SPFDKIM_common_providers>`.
 
-You can check if your DKIM record is valid with a free tool like 
-`DKIM Core <https://dkimcore.org/tools/>`_. If a selector is asked, enter `odoo`.
+Check if the DKIM record is valid with a free tool like `DKIM Core <https://dkimcore.org/tools/>`_.
+If a selector is asked, enter `odoo`.
 
-Check your DMARC policy
-=======================
+Check the DMARC policy
+======================
 
-The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies SPF
-and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
+The Domain-based Message Authentication, Reporting & Conformance (DMARC) is a protocol that unifies
+SPF and DKIM. The instructions contained in the DMARC record of a domain name tell the destination
 server what to do with an incoming email that fails the SPF and/or DKIM check.
```

Comment:
- Replace `SPF and DKIM` with `` :abbr:`SPF (Sender Policy Framework)` and :abbr:`DKIM (DomainKeys Identified Mail)` ``
- Replace `DMARC record` with `` :abbr:`DMARC (Domain-based Message Authentication, Reporting, & Conformance)` record ``
- Replace `SPF and/or DKIM` with `` :abbr:`SPF (Sender Policy Framework)` and/or :abbr:`DKIM (DomainKeys Identified Mail)` ``